### ISSUE TITLE
PHP 8 Support

### ISSUE THREAD

**Issue by sneakingJ** [ivankayzer/howlongtobeat#8]
Hi,
with the newest changes, it seems this package only works with PHP 7. Could you make it compatible with PHP 8.1 too?

**Comment by Askancy** [comment]
use this: https://github.com/Askancy/howlongtobeat

**Comment by sneakingJ** [comment]
> use this: https://github.com/Askancy/howlongtobeat

Oh nice, so everything works still fine except the adjustment with the composer.json?

**Comment by Askancy** [comment]
exactly, the problem is having declared the php version in the composer in the last commit.

```
    "require": {
        "php": "^7.4",
```
